File: tourist-app-backend/src/main/java/ro/touristapp/backend/model/dto/TourAttraction.java
package ro.touristapp.backend.model.dto;

public class TourAttraction {
    private Long id;
    private String name;
    private double latitude;
    private double longitude;
    private String details;

    public TourAttraction(Long id, String name, double latitude, double longitude, String details) {
        this.id = id;
        this.name = name;
        this.latitude = latitude;
        this.longitude = longitude;
        this.details = details;
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public double getLatitude() {
        return latitude;
    }

    public void setLatitude(double latitude) {
        this.latitude = latitude;
    }

    public double getLongitude() {
        return longitude;
    }

    public void setLongitude(double longitude) {
        this.longitude = longitude;
    }

    public String getDetails() {
        return details;
    }

    public void setDetails(String details) {
        this.details = details;
    }
}

File: tourist-app-frontend/src/app/model/TourAttraction.ts
export class TourAttraction {
  id: number;
  name: string;
  latitude: number;
  longitude: number;
  details:string;


  constructor(id: number, name:string, latitude: number, longitude: number, details: string) {
    this.id = id;
    this.name = name;
    this.latitude = latitude;
    this.longitude = longitude;
    this.details = details;
  }
}
